IndexLifecycleRestIT Test Basic Move To Step test failure

failure on `7.x`, does not reproduce locally

logs:
	
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.x+release-tests/519/console
https://gradle-enterprise.elastic.co/s/nyxotdmqku2x4

failure:

```
14:18:58   1> [2019-10-31T09:18:51,555][INFO ][o.e.x.i.IndexLifecycleRestIT] [test] [yaml=ilm/20_move_to_step/Test Basic Move To Step] before test
14:18:58   1> [2019-10-31T09:18:51,862][INFO ][o.e.x.i.IndexLifecycleRestIT] [test] Stash dump on test failure [{
14:18:58   1>   "stash" : {
14:18:58   1>     "body" : {
14:18:58   1>       "indices" : {
14:18:58   1>         "my_index" : {
14:18:58   1>           "index" : "my_index",
14:18:58   1>           "managed" : true,
14:18:58   1>           "policy" : "my_moveable_timeseries_lifecycle",
14:18:58   1>           "lifecycle_date_millis" : 1572531531589,
14:18:58   1>           "age" : "264ms",
14:18:58   1>           "phase" : "warm",
14:18:58   1>           "phase_time_millis" : 1572531531776,
14:18:58   1>           "action" : "forcemerge",
14:18:58   1>           "action_time_millis" : 1572531531776,
14:18:58   1>           "step" : "segment-count",
14:18:58   1>           "step_time_millis" : 1572531531831,
14:18:58   1>           "phase_execution" : {
14:18:58   1>             "policy" : "my_moveable_timeseries_lifecycle",
14:18:58   1>             "phase_definition" : {
14:18:58   1>               "min_age" : "1000s",
14:18:58   1>               "actions" : {
14:18:58   1>                 "forcemerge" : {
14:18:58   1>                   "max_num_segments" : 10000
14:18:58   1>                 }
14:18:58   1>               }
14:18:58   1>             },
14:18:58   1>             "version" : 1,
14:18:58   1>             "modified_date_in_millis" : 1572531531562
14:18:58   1>           }
14:18:58   1>         }
14:18:58   1>       }
14:18:58   1>     }
14:18:58   1>   }
14:18:58   1> }]
14:18:58   1> [2019-10-31T09:18:52,291][INFO ][o.e.x.i.IndexLifecycleRestIT] [test] [yaml=ilm/20_move_to_step/Test Basic Move To Step] after test
14:18:58   2> REPRODUCE WITH: ./gradlew ':x-pack:plugin:ilm:qa:rest:restTestRunner' --tests "org.elasticsearch.xpack.ilm.IndexLifecycleRestIT.test {yaml=ilm/20_move_to_step/Test Basic Move To Step}" -Dtests.seed=63D4A5893B5B0A10 -Dtests.security.manager=true -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dtests.locale=is -Dtests.timezone=America/Matamoros -Dcompiler.java=12 -Druntime.java=8
14:18:58   2> java.lang.AssertionError: Failure at [ilm/20_move_to_step:84]: indices.my_index.step didn't match expected value:
14:18:58              indices.my_index.step: expected String [forcemerge] but was String [segment-count]

```

--- a/x-pack/plugin/ilm/qa/rest/src/test/resources/rest-api-spec/test/ilm/20_move_to_step.yml
+++ b/x-pack/plugin/ilm/qa/rest/src/test/resources/rest-api-spec/test/ilm/20_move_to_step.yml
@@ -73,17 +73,17 @@ teardown:
             action: "complete"
             name: "complete"
           next_step:
-            phase: "warm"
-            action: "forcemerge"
-            name: "forcemerge"
+            phase: "completed"
+            action: "completed"
+            name: "completed"
 
   - do:
         ilm.explain_lifecycle:
           index: "my_index"
   - match: { indices.my_index.policy: "my_moveable_timeseries_lifecycle" }
-  - match: { indices.my_index.step: "forcemerge" }
-  - match: { indices.my_index.action: "forcemerge" }
-  - match: { indices.my_index.phase: "warm" }
+  - match: { indices.my_index.step: "completed" }
+  - match: { indices.my_index.action: "completed" }
+  - match: { indices.my_index.phase: "completed" }
 
 ---
 "Test Invalid Move To Step With Incorrect Current Step":

